bibliopixel/main/all_pixel.py

```diff
@@ -109,3 +109,5 @@ def run(args, settings):
 
 def set_parser(parser):
     parser.set_defaults(run=run)
+    parser.add_argument('--hardware-id', default='1D50:60AB', type=str)
```

Comment:
1. You _must_ add a `help=` to each argument!

2. I _think_ that if you add a default, then `type` defaults to the type of the default?  Have no time to check this, but if you could check that and either remove the `type=` clauses or tell me that my impression was wrong?